manual/managing_github.md

```diff
@@ -0,0 +1,52 @@
+# Numfocus Github permission process
+
+Numfocus defines default permission levels for repositories. The intention is to share as much as we can publicly and restrict access only when required. There are two types of repositories with access restriction. The ones that contains NumFocus board and management sensitive information, the other ones contains data that can’t be made public for other reasons (surveys, tools, etc.).
+
+Access to sensitive repositories must be requested to the board. Access should be reviewed on a quarterly? basis.
+Access to non-sensitive private repository is managed by the ???. 
+
+Approval of access requests are tracked on 
+
+## Numfocus organisation
+
+What is the system to be part of the NF organisation? Do we require contributors to be part of the NF organisation?
+
+## Teams
+
+Teams can only contain people part of the Numfocus organisation on Github.
+
+[1] board and executive directory
+[2] staff
+[3] specific teams for specific usage if required to access private content  (e.g. summit) 
+
+## Repository configuration
+
+Public by default, master branch protection activated (enforce using a PR to contribute)
+
+## Sensitive repositories:
+
+Only one repository seems to quality for sensitive access restriction:
+numfocus-board: only access to the board + executive director
+
+## Private repositories
+
+This list needs to be validated:
+disc-survey: contains survey data, restrict to staff ?? 
+lgl-import
+eventbrite-tools
+evolution
+staff-tools
+NF-projects-DISC-survey
```

Comment:
This is a project generated at the DISC Unconference 2017. I think this can be made public.